packages/react/src/hooks/useAnchoredPosition.ts

```diff
@@ -8,6 +8,7 @@ import useLayoutEffect from '../utils/useIsomorphicLayoutEffect'
 export interface AnchoredPositionHookSettings extends Partial<PositionSettings> {
   floatingElementRef?: React.RefObject<Element>
   anchorElementRef?: React.RefObject<Element>
+  pinPosition?: boolean
```

Comment:
Added this flag so that I'm isolating the new behavior to selectPanel only, I believe we discussed not extending to overlays for now. CC: @camertron